docs/platforms/dotnet/common/troubleshooting.mdx

```diff
@@ -79,4 +79,25 @@ If you're working with obfuscated code, you have several options:
 
 3. **Contribute to Symbolic**: [Our symbolication library is open source](https://github.com/getsentry/symbolic). You could create a ticket to discuss adding support to it with the maintainers.
 
+## Proxy Server
+
+Often, your server can only access the internet through a proxy server.
+If that's the case, make sure your proxy server is configured so the `HttpClient`
+used by Sentry's SDK can pick it up.
+
+```csharp
+(SentryOptions options) =>
+{
+    options.HttpProxy = new WebProxy("read the proxy's address from your Configuration");
```

Comment:
When you [pass a string parameter to the WebProxy constructor](https://learn.microsoft.com/en-us/dotnet/api/system.net.webproxy.-ctor?view=net-9.0#system-net-webproxy-ctor(system-string)), it's supposed to be the URI or address right? So something like:
```csharp
new WebProxy("http://contoso")
```

The sample code above feels a bit misleading... I think either we show some code demonstrating how to read the proxy address from config and then feed that into the `WebProxy` constructor or wereturn new WebProxy("http://contoso");just demonstrate passing in a URI (maybe with a comment saying people need to replace this with the URI of the proxy server on their network).

Reply:
Thanks for the feedback. Changed to a valid `Uri` address.

Do we have a placeholder URL that we commonly use in our docs?